### DISCUSSION TITLE
how to remove this title bar button?

### DISCUSSION THREAD

**Discussion by skydig** [tauri-apps/tauri#10514] [General]
my configure is like following
  "resizable": false,
        "maximized":true,
        "width":1024,
        "height":768,
        "decorations":true
but how to remove following button?
![image](https://github.com/user-attachments/assets/67830c7f-0ebf-4cd7-9855-51573bb2cb18)

**Comment by FabianLars** [comment]
I don't think there's currently a way to do that if `resizable: false` doesn't remove it 🤔 Just to confirm, the button doesn't do anything onclick though right?

**Comment by skydig** [comment]
no, when click it, window shrink to "width":1024,"height":768, click again, maximized,  I just want to keep window maximized, how to do that?

**Reply by FabianLars** [reply]
can you try setting `maximizable: false` (that should only affect the button) and remove the width height configs?

If neither helps, please open an issue because the button is supposed to be disabled with the configs.

**Comment by skydig** [comment]
sir, neither help:(

**Reply by FabianLars** [reply]
Then please open a bug report like i asked :)

https://github.com/tauri-apps/tauri/issues/new?assignees=&labels=type%3A+bug%2Cstatus%3A+needs+triage&projects=&template=bug_report.yml&title=%5Bbug%5D+
